### PR TITLE
Add status for categories

### PR DESCRIPTION
**Pull request by ritamma**

### CODE REVIEW

File: app/controllers/categories_controller.rb [hunk 1]
```diff
@@ -47,4 +48,10 @@ def category_params
   def category_find
     @category = Category.find_by(id: params[:id])
   end
+
+  def for_admin
+    return unless current_user.admin_type == false
```
Comment (fooooxmr) [inline app/controllers/categories_controller.rb:53]: You do not need  == false. You can write unless current_user.admin_type or unless current_user.admin_type?

File: app/controllers/categories_controller.rb [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
-class CategoriesController < ApplicationController
+class CategoriesController < AuthorizationController
```
[the remark this answers is not in the document]
Comment (ritamma) [reply app/controllers/categories_controller.rb:3]: Yes, but I'm not sure how to do it. Do I need to create another controller for guest user or I don't need to use Authorization controller?

**Review by evrusha — APPROVED** [review]

File: app/controllers/categories_controller.rb (hunk 1 above)
Comment (ritamma) [reply app/controllers/categories_controller.rb]: Updated

**Review by Terapool — APPROVED** [review]

File: app/controllers/categories_controller.rb [hunk 3]
```diff
@@ -47,4 +48,12 @@ def category_params
   def category_find
     @category = Category.find_by(id: params[:id])
   end
+
+  def for_admin
```
Comment (fooooxmr) [inline app/controllers/categories_controller.rb:52]: Move this method to Application controller. Or move controller under the Admin namespace.

**Review by fooooxmr — APPROVED** [review]

File: app/controllers/categories_controller.rb (hunk 3 above)
Comment (ritamma) [reply app/controllers/categories_controller.rb]: Updated

**Pull request merged**